dandi/models.py

```diff
@@ -598,71 +621,54 @@ class Participant(DandiBaseModel):
 
     _ldmeta = {
-        "rdfs:subClassOf": ["schema:Thing", "prov:Entity"],
+        "rdfs:subClassOf": ["schema:Person", "prov:Agent"],
         "rdfs:label": "Information about the participant.",
         "nskey": "dandi",
     }
 
 
-class Activity(DandiBaseModel):
-    """Information about the Project activity"""
+class BioSample(DandiBaseModel):
+    """Description about the sample that was studied"""
 
-    identifier: Optional[Identifier] = Field(None, nskey="schema")
-    name: str = Field(
-        title="Title",
-        description="The name of the item.",
-        max_length=150,
-        nskey="schema",
+    identifier: Optional[Identifier] = Field(nskey="schema")
+    altName: Optional[List[Identifier]] = Field(None, nskey="dandi")
+    sampleType: Optional[SampleType] = Field(
+        None, description="OBI based identifier for the sample used", nskey="dandi"
     )
-    description: Optional[str] = Field(
-        None, description="A description of the item.", nskey="schema"
+    assayType: Optional[List[AssayType]] = Field(
+        None, description="OBI based identifier for the assay(s) used", nskey="dandi"
     )
-    startDate: Optional[date] = Field(None, nskey="schema")
-    endDate: Optional[date] = Field(None, nskey="schema")
-
-    # isPartOf: Optional["Activity"] = Field(None, nskey="schema")
-    # hasPart: Optional["Activity"] = Field(None, nskey="schema")
-    wasAssociatedWith: Optional[List[Union[Person, Organization, Software]]] = Field(
-        None, nskey="prov"
+    anatomy: Optional[List[Anatomy]] = Field(
+        None,
+        description="UBERON based identifier for what organ the sample belongs "
+        "to. Use the most specific descriptor.",
+        nskey="dandi",
     )
 
-    schemaKey: Literal["Activity"] = Field("Activity", readOnly=True)
-
-    _ldmeta = {"rdfs:subClassOf": ["prov:Activity", "schema:Thing"], "nskey": "dandi"}
-
-
-# Activity.update_forward_refs()
+    wasDerivedFrom: Optional[List["BioSample"]] = Field(None, nskey="prov")
+    wasAttributedTo: Optional[List[Participant]] = Field(
+        None, description="Participant(s) to which this sample belongs to", nskey="prov"
+    )
+    sameAs: Optional[List[Identifier]] = Field(None, nskey="schema")
+    hasMember: Optional[List[Identifier]] = Field(None, nskey="prov")
 
+    schemaKey: Literal["BioSample"] = Field("BioSample", readOnly=True)
 
-class Project(Activity):
-    name: str = Field(
-        title="Title",
-        description="The name of the project that generated this Dandiset or asset.",
-        max_length=150,
-        nskey="schema",
-    )
-    description: Optional[str] = Field(
-        None, description="A brief description of the project.", nskey="schema"
-    )
-    schemaKey: Literal["Project"] = Field("Project", readOnly=True)
+    _ldmeta = {
+        "rdfs:subClassOf": ["schema:Thing", "prov:Entity"],
+        "rdfs:label": "Information about the biosample.",
```

Comment:
Should we strive to have those consistent with the `__doc__` and thus may be just automagically copied from the class's `__doc__` for every `_ldmeta` (and raise an error if there is no `__doc__`)?  I just don't see why it should differ

Reply:
let's do that in a separate PR. i worry that the doc may end up containing more things than we would want to put into the rdfs:label.